docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that fires when a value appears for the first time in a particular field ({pull}134526[#134526]).
+* Adds related alerts based on process ancestry to alert details ({pull}136009[#136009], {pull}138419[#138419], {pull}135340[#135340]).
+* Adds docs for Osquery API ({pull}137162[#137162]).
+* Creates the "Create dashboard" button to the Dashboards landing page ({pull}136671[#136671]).
+* Adds support for wildcard exceptions for detection rules. New operators are "matches" and "does not match" ({pull}136147[#136147]).
+* Adds a new search query parameter, `dry_run`, to the bulk actions API that allows you to simulate a bulk action without updating rules ({pull}134664[#134664]).
+* Creates the Responder, an interface that enables you to take actions on specific hosts ({pull}134520[#134520])
+* Enables a new method for the Task Manager API: `bulkUpdateSchedules`, which enables you to update the execution timing of `idle` tasks ({pull}132637[#132637]).
+
+[discrete]
+[[bug-fixes-8.4.0]]
+==== Bug fixes and enhancements
+* Updates the Network page's UI to match other overview pages ({pull}137541[#137541], {pull}136913[#136913]).
+* Fixes an error that could occur when you tried to apply an index pattern to rules as a bulk action ({pull}134664[#134664]).
+* Enhances rule previews with configurable rule intervals and look-back times ({pull}137102[#137102]).
+* Enhances the "status pending" badge for endpoint actions with detailed status when you hover on it ({pull}136966[#136966]).
+* Turns grouped navigation on by default ({pull}136819[#136819]).
+* Adds a confirmation dialog to bulk rule export ({pull}136418[#136418]).
+* Adds index pattern info to the Inspect panel({pull}136407[#136407]).
+* Adds a custom dashboards table to the Dashboards page ({pull}136221[#136221]).
+* Fixes a performance issue with alerts that have large fields ({pull}135956[#135956]).
+* Replaces the response actions timeline with the response actions log ({pull}135360[#135360]).
+* Updates the UI for displaying Exceptions ({pull}135255[#135255]).
+* Fixes performance issues with rules management ({pull}135311[#135311]).
+* Allows you to define a fallback `@timestamp` when you've defined a timestamp override ({pull}135116[#135116]).
+* Enhances the Host risk score modal UI ({pull}133708[#133708]).
+* Updates the lists index template to new logic ({pull}133067[#133067]).
+* Adds event filters to the Event Correlation rule type ({pull}132507[#132507]).
+* Allows you to define a data view or index pattern for rules ({pull}130929[#130929]).
+* Adds a treemap and multi-dimensional alert grouping to the Alerts page ({pull}126896[#126896]).
```

Comment:
Flagging this for you @joepeeples

Reply:
This should be in Features since it's a big new change. Here's some suggested rewording:
```suggestion
* Creates a single visualization pane on the Alerts page, and adds a treemap view that shows the distribution of alerts as nested, proportionally-sized tiles. ({pull}126896[#126896]).
```